.github/CODEOWNERS

```diff
@@ -176,6 +176,7 @@ packages/react-components/react-avatar-context @microsoft/teams-prg
 packages/react-components/react-infobutton @microsoft/cxe-red @sopranopillow
 packages/react-migration-v8-v9 @microsoft/cxe-coastal @geoffcoxmsft
 packages/react-components/react-tree @microsoft/teams-prg
+packages/react-components/virtualizer @microsoft/teams-prg
```

Comment:
@Mitch-At-Work should your name be here as well? At least while Virtualizer is new, this will ensure you are looped in on any PRs to this package.

Note: this is not a sneaky push to try to get you to own it long term, if you plan to hand it off after initial contribution and have someone in teams-prg to catch it, that's fine :) It's also fine to have your name here while it's new and feel free to remove it at any point in the future.

Reply:
Sounds good! Will add myself here as well

Reply:
Follow up here, I chatted with @brandonthomas and since his team will be contributing more and more to fluent ui, let's setup his team with a github team here so we can use that in codeowners. 

I created @microsoft/xc-uxe and handed it over to @brandonthomas, we should replace @microsoft/teams-prg with @microsoft/xc-uxe. You can still add @Mitch-At-Work after the team if you want (this is optional). Also, feel free to change the name if you like! I put Brandon on the spot and asked him to give me a name 😆